src/mono/mono/mini/unwind.c

```diff
@@ -258,7 +258,7 @@ decode_uleb128 (guint8 *buf, guint8 **endbuf)
 	guint32 res = 0;
 	int shift = 0;
 
-	while (TRUE) {
+	for (;;) {
```

Comment:
just a personal opinion, but `while (TRUE)` is clearer than `for (;;)`.  I don't think this change is useful.

Reply:
for (;;) is a well-known idiom, and having TRUE in all caps can cause issues with some compilers.